utils/oomparser/oomparser.go

```diff
@@ -144,17 +144,18 @@ func (self *OomParser) StreamOoms(outStream chan *OomInstance) {
 			oomCurrentInstance := &OomInstance{
 				ContainerName: "/",
 			}
-			finished := false
-			for !finished {
+			for line := range lineChannel {
```

Comment:
This is a bit different, since it doesn't check the start line. Is the desired information ever on that first line? If not this is fine, otherwise you can keep the original loop and just check `finished` before receiving the line.